### DISCUSSION TITLE
Feedbacks on the 1st landing page of webcrumbs.org

### DISCUSSION THREAD

**Discussion by jsimples** [webcrumbs-community/webcrumbs#99] [General]
Hi @webcrumbs-community and everyone!

Here's a preview for the 1st landing page of webcrumbs.org:

<a href="https://anysite-2dd7be.webflow.io/" target="_blank">https://anysite-2dd7be.webflow.io/</a>

I'll leave it as it is for a day so you can **share your feedbacks** before it goes live!

See ya!

**Comment by jsimples** [comment]
@shreykoradia @trdxlucifer @dshaw0004 @Ashok-Kumar-dharanikota @WassCodeur @vbrazo @Mannxxx @shubhalgupta @ayobamiseun @rovoskarl @Blossomeze @DaveKobrin @itsayopapi @29deepanshutyagi @maida12

**Reply by dshaw0004** [reply]
That's superb 👌

**Reply by WassCodeur** [reply]
@jsimples Bro I can assure you that you've done a great job. I spent all my time studying the site in the idea of finding some remark but nothing. Bro congratulations, you're a frontend God. I want to do an internship with you.

**Reply by jsimples** [reply]
It took me a while to see these, guys. Thanks!! All comments here were very helpful. We should probably think of how to move forward now with the landing page. Blog and resource contents? What else? Let's move forward! 🚀🚀

**Reply by jsimples** [reply]
I'll open a discussion on frontend next steps too.

**Comment by webcrumbs-community** [comment]
Hey, it's looking good! I'll take a second look and will let you know if I think of anything else.

**Reply by webcrumbs-community** [reply]
I love the copy. Here are minor suggestions: 
- "Open source, free <br/> and full of benefits" instead of "Unlock key benefits that are free and open source".
- "faster than ever" instead of "quicker than ever"
- "plug-and-play" instead of "plug in and play with"
- "For creators and contributors offering plugins, themes, courses, or installing it for clients. " instead of "Create and offer plugins, themes and courses, or install it for your clients."
- "Enter your email to stay updated on the latest tools and tips to manage and scale your React apps effortlessly." instead of "Create, manage, and scale your React apps effortlessly. WebCrumbs is your ticket to a hassle-free, vibrant developer community."

**Reply by jsimples** [reply]
Done!

**Reply by webcrumbs-community** [reply]
Looking at the mobile version right now:

- Just noted that the hamburger menu is empty. Should we add the star our GitHub?
- Some shading on the input would be good for affordance
- Perhaps reducing the size of flags

Just these few comments!

**Reply by jsimples** [reply]
Done!

**Comment by Ashok-Kumar-dharanikota** [comment]
Hi @justicasimples,

It's superb 🚀🤩. The website is very professional and well-crafted.

I'd like to mention one thing about this page.

"The perfect toolkit to elevate your web projects" - perfect.

Regarding "animations," I felt that the font-weight was a tad too large. It seems a bit eye-straining when both the title and the animation content have equal or nearly the same font weights. What are your thoughts on this?

Other than that Everything was great and let's also hear insights from others.

**Reply by jsimples** [reply]
Good eye! I agree! Will change it soon. Thank you!

**Comment by webcrumbs-community** [comment]
If anyone would like to add your own **testimonial**, just let us know. ❤️

**Reply by ayobamiseun** [reply]
assign this to me  😎

**Reply by webcrumbs-community** [reply]
Great! Just drop a line or two of your honest review here and we'll add it to the site. See some examples here: [https://anysite-2dd7be.webflow.io/](https://anysite-2dd7be.webflow.io/)

**Reply by jsimples** [reply]
I'll create a discussion for this!

**Comment by AJBrownson** [comment]
@justicasimples I see you've implemented some nice changes already
I opened the site on my mobile and the sizes of the countries svg flags are very big
You could reduce it to the size it was before or just 0.5rem larger

**Reply by jsimples** [reply]
Agree, @AJBrownson! Thanks for catching it. I've changed it just now.

**Comment by AJBrownson** [comment]
@justicasimples Also it would look better if you put the flags in an auto scrolling carousel just like the one in which those tech companies names are

Looking forward to the finished product 👍

**Reply by jsimples** [reply]
That's a great idea. I changed and I think it's better now!

**Comment by webcrumbs-community** [comment]
Our website has gone live!! 

See it here: https://www.webcrumbs.org/
See also this announcement: https://dev.to/opensourcee/our-website-has-gone-live-1a7

Congratulations everyone!

**Comment by webcrumbs-community** [comment]
Ow, by the way. This is a work in progress, so anytime you have a suggestion, let's keep improving it!
But it's a big step already.
I'm proud of us all.

**Reply by jsimples** [reply]
Yes! See #108 for the next steps

**Comment by AJBrownson** [comment]
@jsimples Fantastic work 👍

**Reply by jsimples** [reply]
Team work. You've spotted some good improvements!

**Comment by mohit-kota** [comment]
@jsimples First of all congrats 👏  on launching your landing page . I see a great interface in the page it's simple and awesome . I see a great scope on this particular project and thinking off to join with you for collaborating from my side .

Thanks and wish you best of luck hope we will connect and work if everything goes good

**Reply by jsimples** [reply]
Really nice, @mohit-kota! Let's work together on the next tasks!
